nl/nl_linux.go

```diff
@@ -45,6 +45,26 @@ var SocketTimeoutTv = unix.Timeval{Sec: 60, Usec: 0}
 // ErrorMessageReporting is the default error message reporting configuration for the new netlink sockets
 var EnableErrorMessageReporting bool = false
 
+// ErrDumpInterrupted is an instance of errDumpInterrupted, used to report that
+// a netlink function has set the NLM_F_DUMP_INTR flag in a response message,
+// indicating that the results may be incomplete or inconsistent.
+var ErrDumpInterrupted = errDumpInterrupted{}
```

Comment:
you could use:
errors.New("results may be incomplete or inconsistent")

Reply:
It is not equivalent: `errDumpInterrupted` implements `interface { Is(error) bool }`. `errors.Is(errors.New(""), unix.EINTR) == false` while `errors.Is(ErrDumpInterrupted, unix.EINTR) == true`.

Reply:
yes you are right.